runtime/oti/jvminit.h

```diff
@@ -452,6 +452,9 @@ enum INIT_STAGE {
 #define VMOPT_XXUSECONTAINERSUPPORT "-XX:+UseContainerSupport"
 #define VMOPT_XXNOUSECONTAINERSUPPORT "-XX:-UseContainerSupport"
 
+#define VMOPT_PORT_VMEM_HUGE_PAGES_MMAP_ENABLED "-XX:+enableHugePagesWithMmap"
+#define VMOPT_PORT_VMEM_HUGE_PAGES_MMAP_DISABLED "-XX:-enableHugePagesWithMmap"
```

Comment:
Naming in this file isn't completely consistent. Sometimes there is 'use', sometimes there is 'enable, sometimes there is nothing (for example for TransparentHugePage). I personally prefer 'nothing' since '+' and '-' serve the purpose of words 'enable' and 'disable'.

Reply:
Cool, so in the case of `VMOPT_PORT_VMEM_HUGE_PAGES_MMAP_DISABLED` it would become `VMOPT_PORT_VMEM_NO_HUGE_PAGES_MMAP`

Reply:
I think having enable/disable in various internal constants and variables are ok. I just did not like it in the name of the command line options.

Reply:
Oh I see what you mean, got it

Reply:
And minor nitpick - command line options should start with capital letters.  ie: `-XX:+Enable...`